libnetwork/drvregistry/networks.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // DriverWalkFunc defines the network driver table walker function signature.
-type DriverWalkFunc func(name string, driver driverapi.Driver, capability driverapi.Capability) bool
+type DriverWalkFunc func(name string, driver driverapi.Driver, capability driverapi.Capability) (stop bool)
```

Comment:
This change is out of scope for this PR

Reply:
Ah, yeah, I had to look up what the return was 😂